test/Passes/DefaultPipeline/linalg-to-loops.mlir

```diff
@@ -0,0 +1,237 @@
+// RUN: tpp-opt %s -default-tpp-passes="linalg-to-loops" -split-input-file | FileCheck %s
+
+// Direct linalg lowering to loops should leave all TPP operations untouched.
+// CHECK-NOT: func.func private @xsmm_
+// CHECK: func.func @tpp_ops(
+// CHECK-SAME:  %[[ARG0:[^ ]+]]: memref<3x3xf32>,
+// CHECK-SAME:  %[[ARG1:[^ ]+]]: memref<3x3xf32>,
+// CHECK-SAME:  %[[ARG2:[^ ]+]]: memref<1x1xf32>)
+func.func @tpp_ops(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>, %arg2: memref<1x1xf32>) {
+  // CHECK: tpp.identity
+  // CHECK: tpp.add
```

Comment:
I wasn't expecting to see tpp ops at all here...

Reply:
It is a test case where we explicitly input TPP ops to check that they remain unchanged when `linalg-to-loops` is enabled.
Just to prevent a regression in case TPP mapping remains disabled but somehow TPP lowering slips into this mode.

Reply:
Makes sense!